packages/amplify-graphql-model-transformer/src/resolvers/generator/rds-vtl-generator.ts

```diff
@@ -0,0 +1,48 @@
+import { ModelDirectiveConfiguration } from "../../directive";
+import { OperationConfig, ModelVTLGenerator } from "./vtl-generator";
+import {
+  generateDefaultLambdaResponseMappingTemplate,
+  generateGetLambdaResponseTemplate,
+  generateLambdaRequestTemplate,
+} from '../rds';
+
+// TODO: This class is created only to show the class structure. This needs a revisit to generate correct resolvers for RDS.
+export class RDSModelVTLGenerator implements ModelVTLGenerator {
+  generateUpdateRequestTemplate(modelName: string, isSyncEnabled: boolean, config?: OperationConfig | undefined): string {
+    return generateLambdaRequestTemplate(modelName, config?.operation!, config?.operationName!);
```

Comment:
Yeah, see my comment below, but the config?.operation! seems like an anti-pattern, our types aren't protecting us here.

Reply:
The arguments typing can be improved as stated in the above comment in a separate PR.